### PR TITLE
Ryan 1711 defaultquerytopostgres

### PR DESCRIPTION
**Pull request by ryandawsonuk**
for Activiti/Activiti#1711

### CODE REVIEW

File: kubernetes/helm/activitiinfrachart/templates/infrastructure.yml [hunk 1]
```diff
@@ -41,6 +42,8 @@ spec:
         env:
         - name: PORT_OFFSET
           value: "22001"
+        - name: DB_VENDOR
+          value: "H2"
```
Comment (erdemedeiros) [inline kubernetes/helm/activitiinfrachart/templates/infrastructure.yml]: Is that normal that you are using H2 here?

File: kubernetes/helm/activitiquerychart/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Activiti 7 Query Service on Kubernetes (see https://github.com/Activiti/)
+name: activitiquerychart
```
Comment (erdemedeiros) [inline kubernetes/helm/activitiquerychart/Chart.yaml]: Is that possible to have a separator or to use camel case for the chart name (activiti-query-chart, activiti_query_chart, activitiQueryChart)? Or is it a convention to use only lower case with no separators ? Same question for the folder name `activitiquerychart `.

Comment (ryandawsonuk) [reply kubernetes/helm/activitiquerychart/Chart.yaml]: yeah we should use hyphens as that is what most do https://github.com/kubernetes/charts/tree/master/stable

File: kubernetes/helm/activitiinfrachart/templates/infrastructure.yml (hunk 1 above)
Comment (ryandawsonuk) [reply kubernetes/helm/activitiinfrachart/templates/infrastructure.yml]: The h2 is embedded (we've been using this all along so far) - the alternative is that we give it a database to use.

File: kubernetes/helm/activitiquerychart/Chart.yaml (hunk 2 above)
Comment (ryandawsonuk) [reply kubernetes/helm/activitiquerychart/Chart.yaml]: I think this is something to discuss separately as it's not part of this change and we're not currently using these more granular charts (which is the bigger question).

**Pull request merged**